econplayground/main/utils.py

```diff
@@ -10,4 +11,12 @@ def user_is_instructor(user):
     except AttributeError:
         instructor_list = INSTRUCTOR_LIST
 
-    return (user.username in instructor_list) or user.is_staff
+    try:
+        instructors_group = Group.objects.get(name='instructors')
+    except Group.DoesNotExist:
+        instructors_group = None
+
+    return (instructors_group and
+            (instructors_group in user.groups.all())) or \
```

Comment:
This could also be handled as
`user.groups.filter(name='instructors').exists()`